xml/repeated-content-decl.ent

```diff
@@ -2144,20 +2140,26 @@ User:           username@ldap.example.com
 '<title xmlns="http://docbook.org/ns/docbook">
   Deploy &kubecf;
  </title>
- <para xmlns="http://docbook.org/ns/docbook">
-  TODO-CAP2 DESCRIPTION
- </para>
  <procedure xmlns="http://docbook.org/ns/docbook">
   <step>
    <para>
-    Use &helm; to deploy &kubecf;.
+    Use &helm; to deploy &kubecf;. Note that you
+    <emphasis role="bold">do not</emphasis> need to manually create the
+    namespace for &kubecf;.
    </para>
-<screen>&prompt.user;helm install <replaceable>kubecf</replaceable> TODO-CHART \
+<screen>&prompt.user;helm install <replaceable>kubecf</replaceable> suse/kubecf \
 --namespace <replaceable>kubecf</replaceable> \
 --values <replaceable>kubecf-values.yaml</replaceable>
 </screen>
   </step>
-  <step><para>2222</para></step>
+  <step>
+   <para>
+    Wait until &kubecf;; is successfully deployed before proceeding. Monitor
+    the status of your &kubecf; deployment using the
+    <command>watch</command> command.
+   </para>
+<screen>&prompt.user;watch --color &apos;kubectl get pods -n <replaceable>kubecf</replaceable>&apos;</screen>
```

Comment:
Should probably use the long version of `-n` like everywhere else.

Reply:
Fixed in https://github.com/SUSE/doc-cap/commit/d33fccaefe1a9941c211a1a821c6da2feedb9a29